### PR TITLE
Fix/time server token

### PR DESCRIPTION
**Pull request by patrickvacek**
Based on the conversation from https://github.com/uptane/deployment-considerations/pull/28#discussion_r339990698.

### CODE REVIEW

**Comment by jhdalek55** [comment]
I'm ok with this, but maybe someone else should look? @trishankkarthik @JustinCappos . Does this make sense?

File: repositories.md [hunk 1]
```diff
@@ -47,10 +47,7 @@ If the Time Server is implemented, the primary is CONDITIONALLY REQUIRED to use
 
 #### ECU Version Report
 
-The ECU version report from each Secondary will contain a token to be sent to the Time Server. To prevent a replay, this token SHOULD be unique for each update cycle. As we expect that these updates will be relatively infrequent (e.g., due to a limited number of write cycles), there will be a sufficient number of tokens to make this possible.
-
-
-The payload of the ECU version report sent to the Director may contain the token sent to the Time Server. This token is part of the version report sent from Secondaries to the Primary. If the token is removed, the signature will not match.
+The ECU version report from each Secondary will contain a token to be sent to the Time Server. To prevent a replay, each token SHOULD be unique per ECU for each update cycle. As we expect that these updates will be relatively infrequent (e.g., due to a limited number of write cycles), there will be a sufficient number of tokens to make this possible. The payload of the ECU version report sent from the Primary to the Director MAY contain the tokens sent to the Time Server. In this case, if any token is removed or changed, the signature will not match.
```
Comment (JustinCappos) [inline repositories.md]: As I understand https://github.com/uptane/deployment-considerations/pull/28#discussion_r339990698, one could rephrase this to indicate "One MUST send the tokens.  These tokens MAY be included in the ECU version report."  Is this accurate?   I think we have the second case covered, but not the first one.

Comment (patrickvacek) [reply repositories.md]: Isn't this already made clear in the preceding section? This is just describing how you can send them, but the requirement that they must be sent was already described.

Comment (JustinCappos) [inline repositories.md]:
```suggestion
The ECU version report from each Secondary will contain a token to be sent to the Time Server in whatever manner the implementer chooses.  For example, the payload of the ECU version report sent from the Primary to the Director MAY contain the tokens sent to the Time Server. In this case, if any token is removed or changed, the signature will not match.  To detect a replay attack, each token SHOULD be unique per ECU. As we expect that these updates will be relatively infrequent (e.g., due to a limited number of write cycles), there will be a sufficient number of tokens to make this possible.  
```

Two changes here.  First, I tried to make it clearer that this was an option, but that sending was required.   Second, I tried to clarify that tokens need to be unique, not just unique per this "cycle" (which seems to mean per update and isn't well defined).

I expect this won't be taken as is, but wanted to use this to try to start more conversation...

Comment (patrickvacek) [reply repositories.md]: Actually, this looks great to me. Thanks!

Comment (patrickvacek) [reply repositories.md]: I think Justin's suggestion made this more clear. The idea is that you could send the tokens in a different way, but if you do it this way, you get the benefit of signature mismatches if something goes wrong. Is it clearer now that Justin's suggestion is merged?

**Comment by jhdalek55** [comment]
Are we good to merge then?

File: repositories.md (hunk 1 above)
Comment (jhdalek55) [reply repositories.md]: Yes. I think this is good to go. Should I merge?

Comment (patrickvacek) [reply repositories.md]: Up to you! I'm happy with it.

Comment (jhdalek55) [reply repositories.md]: OK. I'm merging.
